docs/src/app/components/pages/components/BottomNavigation/ExampleSimple.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import FontIcon from 'material-ui/FontIcon';
+import {BottomNavigation, BottomNavigationItem} from 'material-ui/BottomNavigation';
+import Paper from 'material-ui/Paper';
+
+const recentsIcon = <FontIcon className="material-icons">restore</FontIcon>;
+const favoritesIcon = <FontIcon className="material-icons">favorite</FontIcon>;
+const nearbyIcon = <FontIcon className="material-icons">location_on</FontIcon>;
+
+class BottomNavigationExampleSimple extends React.Component {
```

Comment:
Regarding the syntax of the example. We now officialy use the `stage-1` on approach going forward.
I would be great to update the example to follow it too. You have an example on the `next` branch.
https://github.com/callemall/material-ui/blob/next/docs/site/src/demos/menus/SimpleMenu.js